docs/pages/guide/[slug].tsx

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs';
+import matter from 'gray-matter';
+import { marked } from 'marked';
+import { useRouter } from 'next/router';
+import path from 'path';
+
+interface Props {
+  slug: string;
+  frontmatter: { title: string; href: string; description: string };
+  content: string;
+}
+
+const PackagePage: React.FC<Props> = ({ frontmatter: { title }, content }) => {
+  const router = useRouter();
+
+  return (
+    <>
+      <div style={{ color: 'blue' }}>
+        <div onClick={() => router.back()}>&larr; Go Back</div>
+      </div>
+      <h3>{title}</h3>
+      <div dangerouslySetInnerHTML={{ __html: marked(content) }}></div>
+    </>
+  );
+};
+
+export async function getStaticPaths() {
+  const files = fs.readdirSync(path.join('guide'));
+  const paths = files.map(filename => ({ params: { slug: filename.replace('.md', '') } }));
+  return { paths, fallback: false };
+}
+export async function getStaticProps({ params: { slug } }: { params: { slug: string } }) {
+  console.log(slug);
+  const markDownWithMeta = fs.readFileSync(path.join('guide', `${slug}.md`), 'utf-8');
+  const { data: frontmatter, content } = matter(markDownWithMeta);
```

Comment:
This also needs decoding.

Also, this looks similar to the `getStaticProps` from the main guide component. Could we extract the common stuff? Maybe create a `getAllGuides` method.